app/javascript/packs/containers/ReviewModeContainer.jsx

```diff
@@ -7,10 +7,89 @@ import React from 'react';
 
 import QuestionContainer from './QuestionContainer';
 
+import { getAnswerForQuestionAndBuilding } from '../selectors/answersSelector';
+import { getDependentQuestionsForOption } from "../selectors/questionsSelector";
 import { connect } from 'react-redux';
 import { getQuestionsByBuilding } from '../selectors/questionsSelector';
 
+import { post, patch } from '../fetch/requester';
+
+async function postDelegations(delegations, callback) {
+  var body = {
+    "delegations": delegations,
```

Comment:
Style nit: keys don't need quotes if they don't have disallowed chars. Since key and value share a name here, this can also be simply `{ delegations }`